idseq_dag/util/m8.py

```diff
@@ -181,21 +183,49 @@ def accumulate(hits, accession_id):
                 # provided by other accessions. This occurs a lot and
                 # handling it in this way seems to work well.
                 continue
-            hits[level][taxid_at_level] = accession_id
+            accession_list = hits[level].get(taxid_at_level, []) + [accession_id]
+            hits[level][taxid_at_level] = accession_list
         return blacklisted
 
+    def most_frequent_accession(accession_list):
+        counts = Counter(accession_list)
+        return counts.most_common(1)[0][0]
+
+
     def call_hit_level(hits, blacklisted = False):
-        ''' Call hit if read not blacklisted and only one taxid at level '''
+        ''' TO BE DEPRECATED: Call hit if read not blacklisted and only one taxid at level '''
         if not blacklisted:
             for level, hits_at_level in enumerate(hits):
                 if len(hits_at_level) == 1:
-                    taxid, accession_id = hits_at_level.popitem()
+                    taxid, accession_list = hits_at_level.popitem()
+                    accession_id = most_frequent_accession(accession_list)
```

Comment:
Previously we just picked an arbitrary accession from that list.    I suppose there could be multiple occurrences of the same accession id, if the read maps equally well to different regions of the reference sequence?